bcrs-ui/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "hello-world",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test:unit": "vue-cli-service test:unit --testPathPattern"
+  },
+  "dependencies": {
+    "@mdi/font": "^4.5.95",
+    "axios": "^0.18.0",
+    "core-js": "^3.4.3",
+    "sbc-common-components": "2.0.22",
+    "vue": "^2.6.10",
+    "vue-class-component": "^7.0.2",
+    "vue-property-decorator": "^8.3.0",
+    "vue-router": "^3.1.3",
+    "vuetify": "^2.0.11",
+    "vuex": "^3.1.2",
+    "vuex-class": "^0.3.2"
+  },
+  "devDependencies": {
```

Comment:
I'd love to see this list ordered alphabetically :)
(easier to compare with other projects' lists)

Reply:
Yes sir, great idea, you got it!